libs/react/src/lib/select/select.tsx

```diff
@@ -1,121 +1,98 @@
-import React, { ReactNode } from 'react'
+import React, { useState } from 'react'
 import {
-  IndicatorType,
+  IExpandableInformation,
+  ILabelAndLabelInformation,
   IValidator,
   randomId,
   validateClassName,
 } from '@sebgroup/extract'
 import { ChevronDown } from '../icons'
-import { SelectorAttributesProps, LabelProps } from '../../types'
 import { FormItem } from '../formItem'
+import classNames from 'classnames'
 
-export interface SelectProps extends SelectorAttributesProps, LabelProps {
-  children: ReactNode
-  validator?: IValidator
-  onChange?: (event: React.ChangeEvent<HTMLSelectElement>) => void
-  onSelect?: (event: React.ChangeEvent<HTMLSelectElement>) => void
-  defaultValue?: string | number
-  value?: string | number
-  testId?: string
-  expandableInfo?: string
-  expandableInfoButtonLabel?: string
-  name?: string
-}
-
-interface OptionProps {
-  children: string
-  value: string | number
-  disabled?: boolean
-  invalid?: boolean
-  autocomplete?: string
-  autofocus?: boolean
-  form?: string
-  name?: string
-  required?: boolean
-  size?: number
-  selected?: boolean
-}
-
-interface OptionGroupProps {
+export interface SelectProps 
```

Comment:
use the react props definitions for select instead of defining everything manually